extensions/sql-bindings/src/services/azureFunctionsService.ts

```diff
@@ -159,6 +165,18 @@ export async function createAzureFunction(node?: ITreeNodeInfo): Promise<void> {
 					newNode = newNode.parentNode;
 				}
 			}
+
+			// Prompt user for binding type
+			telemetryStep = CreateAzureFunctionStep.getBindingType;
```

Comment:
You currently have a telemetry step for `launchFromTable` which isn't going to be accurate after this change. Would suggest making it something more generic like `launchFromObjectExplorer` and then include an `itemType` or something with the type of node being launched from.